x11-packages/shotcut/build.sh

```diff
@@ -0,0 +1,15 @@
+TERMUX_PKG_HOMEPAGE=https://shotcut.org/
+TERMUX_PKG_DESCRIPTION="Cross-platform Qt based Video Editor"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux-user-repository"
+TERMUX_PKG_VERSION=24.04.28
+TERMUX_PKG_SRCURL=https://github.com/mltframework/shotcut/archive/refs/tags/v$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=cc40aeb8b7f50fb8e102f45e1accbc4b4153f14b565154a4f4a6ec446f5591e4
+# TODO: movit (not avialable)
+TERMUX_PKG_DEPENDS="ffmpeg, fftw, frei0r-plugins, mlt, libx264, libvpx, lame, ladspa-sdk, qt6-qtbase, qt6-qtcharts, qt6-qtdeclarative, qt6-qtimageformats, qt6-qtmultimedia, qt6-qttranslations, sdl, sdl2"
+TERMUX_PKG_BUILD_DEPENDS="qt6-qttools"
+TERMUX_PKG_AUTO_UPDATE=true
+# TODO: remove the Debug build type after finish this pacakge
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+-DCMAKE_BUILD_TYPE=Debug
+"
```

Comment:
32bit builds work when setting `-Wno-c++11-narrowing`, not sure if the resulting package is functional though.
```suggestion
"

termux_step_pre_configure(){
	CXXFLAGS+=' -Wno-c++11-narrowing'
}
```